ubuntudesign/docs.vanillaframework.io
# Updated vanilla-docs-theme to v1.5.3

## Done

Updated embedded vanilla-docs-theme to v1.5.3.

## QA

- Pull the branch
- Run `./build-html`
- Run `caddy`
- Visit http://127.0.0.1:8543/en/ and ensure everything looks how it should.

## Details

Fixes https://github.com/ubuntudesign/docs.vanillaframework.io/issues/94
